docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["digitalio", "busio"]
```

Comment:
I glanced at the library, doesn't seem like it imports `digitalio`?

Reply:
I removed this, but it has nothing to do with pypi, only sphinx. You're right, the docs are no longer using it as a referenced library.